lib/application/join_home_by_id/join_home_by_id_bloc.dart

```diff
@@ -46,7 +47,7 @@ class JoinHomeByIdBloc extends Bloc<JoinHomeByIdEvent, JoinHomeByIdState> {
     } else {
       final AllHomesOfUserEntity allHomesOfUserEntity = AllHomesOfUserEntity(
         id: AllHomesOfUserUniqueId.fromUniqueString(event.id),
-        name: AllHomesOfUserName('home'),
+        name: AllHomesOfUserName('home'.tr()),
```

Comment:
I think it is better to remove this change entirely because it will require testing.